wordpress-mobile/gutenberg-mobile
# Paragraph block accessibility improvements - unselected state

Fixes part of https://github.com/wordpress-mobile/gutenberg-mobile/issues/907

Changes reside in gutenberg PR: https://github.com/WordPress/gutenberg/pull/15126

**To test (iOS):**

Activate VoiceOver: https://github.com/wordpress-mobile/WordPress-iOS/wiki/Using-VoiceOver

**To test (Android):**

Activate TalkBack: https://www.wikihow.com/Enable-TalkBack-on-Your-Android
In my case, the system had a handy tutorial on how to use it.
If your version doesn't have it, it's actually very similar to VoiceOver.

**Test Steps**

Tap on an empty paragraph block
You should hear "Paragraph block. Empty"

Tap on a filled paragraph block
You should hear "Paragraph block. <text content>"
